tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -0,0 +1,559 @@
+            moc_front_end.add_extension(TelemetryExtension("mo", t.send_event, t.send_error, t.send_stack_trace))
+            moc_front_end.add_extension(ProgressReporterExtension(progress_printer(argv)))
+            if legacy_transformations_config_used(argv):
+                raise Error('Legacy extensions are not supported for the new frontend')
+            if legacy_extensions_used(argv):
+                raise Error('Legacy transformations configuration is not supported for the new frontend')
+            if new_transformations_config_used(argv):
+                moc_front_end.add_extension(JsonConfigExtension(argv.transformations_config))
+            if new_extensions_used(argv):
+                for extension in argv.extensions:
+                    moc_front_end.add_extension(extension)
+            ngraph_function = moc_pipeline(argv, moc_front_end)
+
+            return graph, ngraph_function
+        else:  # apply fallback
+            reasons_message = ", ".join(fallback_reasons)
+            load_extensions(argv, *list(deduce_legacy_frontend_by_namespace(argv)))
+            t.send_event("mo", "fallback_reason", reasons_message)
+            log.warning("The IR preparation was executed by the legacy MO path. "
+                        "This is a fallback scenario applicable only for some specific cases. "
+                        f"The detailed reason why fallback was executed: not supported {reasons_message} were used. "
+                        "You can specify --use_new_frontend flag to force using the Frontend MO path to avoid additional checks. " +
+                        refer_to_faq_msg(105))
+
+    t.send_event("mo", "conversion_method", "mo_legacy")
+    graph = unified_pipeline(argv)
+
+    return graph, ngraph_function
+
+
+def emit_ir(graph: Graph, argv: argparse.Namespace):
+    NormalizeTI().find_and_replace_pattern(graph)
+    for_graph_and_each_sub_graph_recursively(graph, RemoveConstOps().find_and_replace_pattern)
+    for_graph_and_each_sub_graph_recursively(graph, CreateConstNodesReplacement().find_and_replace_pattern)
+
+    if 'feManager' in argv:
+        del argv.feManager
+
+    mean_data = deepcopy(graph.graph['mf']) if 'mf' in graph.graph else None
+    input_names = deepcopy(graph.graph['input_names']) if 'input_names' in graph.graph else []
+
+    prepare_emit_ir(graph=graph,
+                    data_type=graph.graph['cmd_params'].data_type,
+                    output_dir=argv.output_dir,
+                    output_model_name=argv.model_name,
+                    mean_data=mean_data,
+                    input_names=input_names,
+                    meta_info=get_meta_info(argv),
+                    use_temporary_path=True)
+
+    # This graph cleanup is required to avoid double memory consumption
+    graph.clear()
+
+    output_dir = argv.output_dir if argv.output_dir != '.' else os.getcwd()
+    orig_model_name = os.path.normpath(os.path.join(output_dir, argv.model_name))
+
```

Comment:
why did we miss that check `if not (argv.framework == 'tf' and argv.tensorflow_custom_operations_config_update)` here?

Reply:
Returned this check as discussed offline.